pkg/sql/opt/testutils/testcat/create_table.go

```diff
@@ -44,6 +44,10 @@ const (
 
 var uniqueRowIDString = "unique_rowid()"
 
+// creteFKIndexes controls whether we automatically create indexes on the
+// referencing side of foreign keys (like it was required before 20.2).
+const createFKIndexes = false
```

Comment:
Is there a reason you want to keep this code around behind a flag rather than just removing it?

Reply:
It can be removed later, there's no downside to keeping it. I'm thinking we may want to temporarily enable it if we want to see how a testcase would change in 20.1 without backporting everything first.